constants/Dictionary.ts

```diff
@@ -401,6 +401,72 @@ const MESSAGES: any = {
         youMustAllowTheLocationPermissionForAMoreAccurateResult:    'आपको अधिक सटीक परिणाम के लिए स्थान की अनुमति देनी होगी',
         later:                                                      'बाद मे'
     },
+    tr:{ 
+    add:                                                        'ekle',
+    perishableGoods:                                            'ikinci el mallar',
+    setLocation:                                                'lokasyon olustur',
+    bestBefore:                                                 'best before',
+    successfullyAdded:                                          'basarıyla eklendi',
+    letsContinueWithOtherPerishableGood:                        "hadi diğer ikinci el mallarla devam edelim",
+    cancel:                                                     'iptal et',
+    okay:                                                       'tamam',
```

Comment:
The next words start with upper case in the English translation: `okay, new, all/mine, nearby`. (We cannot dynamically capitalize these during their time of use because of context.) Do you do the same in Turkish as the English to start the sentences with capital letter? If you do then please transform those, otherwise it is good!

Reply:
I did mean like [this](https://github.com/ivgraai/expiry-util/commit/c293178fa5a163a0ea7a3a93505efa35d561cf4f). But it is now merged! :wink: